### PR TITLE
Enable Google one tap for super users

### PR DESCRIPTION
**Pull request by indeewari**
### Proposed changes in this pull request

Google one tap enabling checkbox at carbon console

#### Security

- [ ] **Confirm this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.**
- [ ] **Are all UI and API inputs run through forms or serializers?**
- [ ] **Are all external inputs validated and sanitized appropriately?**
- [ ] **Does all branching logic have a default case?**
- [ ] **Does this solution handle outliers and edge cases gracefully?**
- [ ] **Are all external communications secured and restricted to SSL?**

#### Documentation

- [ ] **Are changes to the UI documented in the platform docs?** If this PR introduces new platform site functionality or changes existing ones, the changes should be documented.
- [ ] **Are changes to the API documented in the API docs?** If this PR introduces new API functionality or changes existing ones, the changes must be documented.
- [ ] **Are reusable components documented?** If this PR introduces components that are relevant to other developers (for instance a mixin for a view or a generic form) they should be documented in the Wiki.

### CODE REVIEW

**Review by somindatommy — APPROVED** [review]

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp [hunk 1]
```diff
@@ -4988,6 +4988,21 @@
                                 <td class="leftCol-med labelField"><%=prop.getDisplayName()%>:</td>
                                 <%} %>
                                 <td>
+
+                                <% if (StringUtils.isNotEmpty(prop.getType()) && prop.getType().equalsIgnoreCase("boolean")) {
+                                    String propChecked = "";
+                                    if ( prop.getValue() != null && prop.getValue().equalsIgnoreCase("true")){
```
Comment (ShanChathusanda93) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp:4994]:
```suggestion
                                    if (prop.getValue() != null && prop.getValue().equalsIgnoreCase("true")){
```

Comment (ShanChathusanda93) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp:4994]: Is prop.getValue() returns a String? If so shall we use the StringUtils?

Comment (indeewari) [reply components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]: This is fixed e983cb8c16466396376a0234e2c1c0aff219187d

Comment (indeewari) [reply components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]: This is fixed e983cb8c16466396376a0234e2c1c0aff219187d

**Review by somindatommy — APPROVED** [review]

**Pull request merged**